include/functions.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+function debugOutput($input){
+    echo "<pre>";
+    var_dump($input);
+    echo "</pre>";
+}
+
+
+function convertDateToMDY($dateToConvert) {
+    $datePosted = $dateToConvert;
```

Comment:
You don't necessarily need this line -- I would just pass $dateToConvert to DateTime() and then save it to $datePosted